src/emulator/audio/src/audio.cpp

```diff
@@ -103,7 +92,7 @@ bool init(AudioState &state, ResumeAudioThread resume_thread) {
     desired.freq = 48000;
     desired.format = AUDIO_S16LSB;
     desired.channels = 2;
-    desired.samples = 512;
+    desired.samples = 1024;
```

Comment:
This should ideally be configurable right? Unrelated to this PR, just an observation.

I remember I had to do this for decaf (https://github.com/decaf-emu/decaf-emu/pull/397) since games behaved differently on different hardware and there's a trade-off between audio latency/stuttering.

Reply:
This is a good shout. I split that change into a separate commit and added a note to the commit message.